omnibus-snaxchain.toml

```diff
@@ -0,0 +1,26 @@
+name = "synthetix-omnibus"
+version = "1"
+description = "Snaxchain deployment"
+include = [
+    # Governance
+    "tomls/governance/councils-mothership.toml",
+    "tomls/governance/wormhole.toml",
+]
+
+[setting.owner]
```

Comment:
May I suggest to use separate 3 settings?
- owner
- deployer
- pdao

We do it for other deployments and there is a benefit of doing it, as all three have different purpose

Reply:
Updated in #370, looks like there is no need to use pdao, and it does not look like there is a use for deployer. But I have added `deployer` setting anyway.